### DISCUSSION TITLE
Con l'integrazione Smartir riesco a controllare i climatizzatori (SAMSUNG) ma non un televisore

### DISCUSSION THREAD

**Discussion by rmoggia** [indomus/forum#564] [HUB: Home Assistant]
Buongiorno. 
Nel dettaglio ho installato l'integrazione Smartir. Controllo i climatizzatori bene con Broadlink. Ma non un televisore. Il televisore è un Telesystem per il quale ho catturatori tutti codice e costruito il relativo JSON e localizzato sotto la cartella:
`config\custom_components\smartir\codes\media_player`.  Ho fatto un test a utilizzare invece di smartir uno swith riesco ad accendere e spegnere il televisore  Allego i files di configurazione. 
[configuration.yaml.txt](https://github.com/indomus/forum/files/6878022/configuration.yaml.txt)
[8000.json.txt](https://github.com/indomus/forum/files/6878019/8000.json.txt)
Alla ripartenza di smartir mi dovrei trovare in lovelace il televisore fra i media players ma niente.
Se qualcuno avesse dei suggerimenti mi farebbe cosa gradita giacchè sto impazzendo. 
Grazie in anticipo

**Comment by indomus** [comment]
Controlla i log che dicono quando sale HA, qualcosa ti dirà. Avrai sbagliato qualcosa nella configurazione.

**Reply by rmoggia** [reply]
Ciao. Grazie dell'attenzione.
In effetti il problema sembra essere il json che ho preparato perchè con un altro televisore i cui codici sono già disponibili funziona nel senso che dall'interfaccia lovalace riesco ad accendere e spegnerlo ma non altro. Però non ho approfondito. Vi aggiorno.

**Reply by rmoggia** [reply]
Il problema era nei nomi assegnati ai tasti che debbono essere conformi ai json di altri telecomandi.
Adesso funzionano l'accensione e spegnimenti, volume su e giu e mute. 
Grazie a tutti per l'attenzione
